draft-ietf-httpbis-message-signatures.md

```diff
@@ -380,9 +381,67 @@ Given the following signature parameters:
 The signature parameter value is defined as:
 
 ~~~
-"@signature-params": ("@request-target" "host" "date" "cache-control" "x-empty-header" "x-example" "x-dictionary";key=b "x-dictionary";key=a "x-list";prefix=3); keyid="test-key-a"; alg="hs2019"; created=1402170695; expires=1402170995
+# NOTE: '\' line wrapping per RFC 8792
+"@signature-params": ("@request-target" "host" "date" "cache-control" "x-empty-header" \
+   "x-example" "x-dictionary";key=b "x-dictionary";key=a "x-list";prefix=3); 
+   \keyid="test-key-a"; alg="rsa-pss-SHA256"; created=1402170695; expires=1402170995
 ~~~
 
+## Create the Signature Input {#create-sig-input}
```

Comment:
When rewriting this I found the previous structure to be too much abstraction to make sense of the different pieces. I moved this section up because it follows along with what the rest of the section is about: creating the signature input. I think we can manage this with changing the intro text to section 2 instead.